lib/misc.py

```diff
@@ -45,5 +45,15 @@ def list_(cli):
    for img in sorted(imgs):
       print(ch.Image_Ref(img))
 
+def reset(cli):
```

Comment:
Couple suggestions:

1. I'd suggest a single `rmtree` that just removes the builder storage directory. Then you don't have to enumerate the contents somehow.
2. Check to make sure it really looks like a builder storage directory. Not looking right is a consistency problem; either we messed up or they have the storage directory path wrong. If it's not there, that path could also be wrong, but also you could get that if you just `ch-image reset` twice in a row; in that case the error could be something like "no storage directory found; misconfiguration or already reset?" (but write it better).
3. I'd use `FATAL` here even though `ch-image` exits immediately after the method.